tools/pyang_plugins/gobgp.yang

```diff
@@ -889,6 +889,14 @@ module gobgp {
 
   }
 
+  augment "/rpol:routing-policy/rpol:policy-definitions/" +
```

Comment:
Don't you have to update this as well? https://github.com/osrg/gobgp/blob/master/api/gobgp.proto#L1076

I tried rebuilding my grpc bindings for python and the following didn't work:

```
>>> import gobgp_pb2
>>> policy = gobgp_pb2.Policy()
>>> policy.name = "a_policy_that_uses_go_code"
>>> policy.plugin_path = "/path/to/policy.go"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'Policy' object has no attribute 'plugin_path'
```

Or am I missing something?

Reply:
Sorry, I haven't yet consider the implementation for the gRPC API, I did just "by using configuration file" case. I will investigate what are required to support this feature on the gRPC API and update PR.